easybuild/easyblocks/o/openblas.py

```diff
@@ -55,7 +55,11 @@ def build_step(self):
             del os.environ[cflags]
             self.log.info("Environment variable %s unset and passed through command line" % cflags)
 
-        cmd = "%s make %s %s" % (self.cfg['prebuildopts'], ' '.join(build_parts), self.cfg['buildopts'])
+        paracmd = ''
+        if self.cfg['parallel']:
+            paracmd = "-j %s" % self.cfg['parallel']
+
+        cmd = ' '.join([self.cfg['prebuildopts'], 'make', paracmd, ' '.join(build_parts), self.cfg['buildopts']])
```

Comment:
Sorry for the nitpicking, but having a variable called `paracmd` that has no command in it might be confusing, I would suggest the following
```
        makecmd = 'make'
        if self.cfg['parallel']:
            makecmd = "%s -j %s" % (makecmd, self.cfg['parallel'])

        cmd = ' '.join([self.cfg['prebuildopts'], makecmd, ' '.join(build_parts), self.cfg['buildopts']])
```